internal/wrappers/mock/projects-mock.go

```diff
@@ -40,6 +40,54 @@ func (p *ProjectsMockWrapper) Get(params map[string]string) (
 		filteredTotalCount = 0
 	}
 
+	if params["names"] == "fake-kics-scanner-fail" {
```

Comment:
Suggestion -you can change all this makes to switch case and a method the creates the fake with params